tcms/testruns/tests/test_views.py

```diff
@@ -565,3 +565,157 @@ def test_show_add_cases_to_run(self):
             ]
             for html in html_pieces:
                 self.assertContains(response, html, html=True)
+
+
+class TestCaserunEditMenus(BaseCaseRun):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
```

Comment:
Method that only calls `super()` is useless. It doesn't do anything on its own and directly delegates to the parent class as if it didn't exist.